File: backend/src/main/java/com/matheus/voting_session_api/votingsession/dto/response/VotingSessionResponse.java
package com.matheus.voting_session_api.votingsession.dto.response;

import java.time.Instant;

public record VotingSessionResponse(
        Long id,
        String topic,
        String description,
        Instant startAt,
        Instant endAt,
        boolean isActivated,
        VotingSessionInfo votingSessionInfo
) { }

File: frontend/src/app/voting-session/model/response/voting-session-response.ts
import { VotingSessionInfo } from "./voting-session-info";

export interface VotingSessionResponse {
  id: number;
  topic: string;
  description: string;
  startAt: Date;
  endAt: Date;
  isActivated: boolean
  votingSessionInfo: VotingSessionInfo;
}
